libs/application/templates/inheritance-report/src/fields/EstateAndVehiclesRepeater/FieldComponent.tsx

```diff
@@ -0,0 +1,103 @@
+import { InputController } from '@island.is/shared/form-fields'
+import { useLocale } from '@island.is/localization'
+import { m } from '../../lib/messages'
+import DoubleColumnRow from '../../components/DoubleColumnRow'
+import { parseLabel } from '../../lib/utils/helpers'
+import ShareInput from '../../components/ShareInput'
+import { RealEstateLookup } from './RealEstateLookup'
+import { VehicleLookup } from './VehicleLookup'
+import { FieldComponentProps } from './types'
+
+export const FieldComponent = ({
+  assetKey,
+  onAfterChange,
+  setLoadingFieldName,
+  loadingFieldName,
+  pushRight,
+  field,
+  fieldIndex,
+  fieldName,
+  error,
+  answers,
+  readOnly,
+  disabled,
+}: FieldComponentProps) => {
+  const { formatMessage } = useLocale()
+
+  const defaultProps = {
+    ...field,
+    id: fieldName,
+    name: fieldName,
+    format: field.format,
+    label: formatMessage(parseLabel(field.title, answers)),
+    defaultValue: '',
+    type: field.type,
+    placeholder: field.placeholder,
+    backgroundColor: field.color ? field.color : 'blue',
+    currency: field.currency,
+    required: readOnly ? false : field.required,
+    loading: fieldName === loadingFieldName,
+    suffix: field.suffix,
+    onChange: () => onAfterChange?.(),
+    error: error,
+    readOnly: readOnly,
+    disabled: disabled,
+  }
+
+  let content = null
```

Comment:
Does content have to be null?
What about just declaring the variable since we have an override in the default case of the switch? 
```suggestion
  let content
```